roles/cdh/templates/nifi.j2

```diff
@@ -0,0 +1,32 @@
+{
+  "refName": "NIFI-1",
+  "serviceType": "NIFI",
+  "serviceConfigs": [
+    {
+      "name": "kerberos.auth.enabled",
+      "value": "{{ (krb5_kdc_type == 'ad') or (krb5_kdc_type == 'mit') | lower }}"
+    }, {
+      "name": "nifitoolkitca_service",
+      "ref": "NIFITOOLKITCA-1"
+    }, {
+      "name": "zookeeper_service",
+      "ref": "ZOOKEEPER-1"
+    }
+  ],
+  "roleConfigGroups": [
+    {
+      "refName": "NIFI-1-NIFI_NODE-BASE",
+      "roleType": "NIFI_NODE",
+      "configs" : [ 
+        {
+          "name" : "nifi.toolkit.tls.regenerate",
+          "value" : "true"
+        }, {
+          "name" : "ssl_enabled",
+          "value" : "true"
```

Comment:
maybe also use a var to enable ssl, like 
`       "variable": "NIFI_SSL_ENABLED" ` ?